docs/sources/reference/templating.md

```diff
@@ -386,6 +386,15 @@ This variable is the UID of the current dashboard.
 This variable is the ID of the current organization.
 `${__org.name}` is the name of the current organization.
 
+### The $__user Variable
+> Only available in Grafana v6.8+
+
+This variable is the ID of the current user.
+`${__org.name}` is the name of the current user.
+`${__org.login}` is the login of the current user.
+`${__org.email}` is the email of the current user.
```

Comment:
Should this be changed to `__user.*` instead of the `__org.*`?

Reply:
no, think its good that the options here are listed

Reply:
Yea - that copy/paste at 4am strikes again...

Have updated the process so that it no longer relies on the properties being pulled into contextSvr from the user record - and rather pulls the info directly, fixed documentation, confirmed typecheck passes and integrated change into second declaration location.

Thanks for your input

Reply:
Torkelo - mckn is correct re the __user*  instead of __org* in the documentation it should be __user.*   - If you would like me to expand this change to make available all properties of the user object - am happy to do so.

Reply:
I think it looks really good @aidanmountford!  Maybe also add the `__org.id` in the section about the `organizations`?

*Optional:* if you want to make it even better you can convert the (completeDashboardTemplating-thunk)[https://github.com/grafana/grafana/blob/6a5794aa7a44422c7c1e64d337dad33fab946e52/public/app/features/variables/state/actions.ts#L87] to `async/await` and instead of using a promise chain you can `await` the result from the `backendSrv`.

We can then also change so the (dispatch)[https://github.com/grafana/grafana/blob/c7ffc1199cdb331c34cf8d95a60ed602b14e76b5/public/app/features/dashboard/state/initDashboard.ts#L190] is being awaited as with the `processVariables` on the live above.

Reply:
I noticed that the tests are failing, please give me a ping if you need help sorting those out.

Reply:
mckn :  I have moved the backend server call to an await before I read this for several reasons, one of which it makes the spec tests sort themselves out, and two, because it will be easier for the next person to simply duplicate the database request(s) and add a few lines of code to expand this.

I will expand the org, dashbaord and user objects to include all useful fields while I'm here - any other objects you believe might be worth exposing  -  I will do them at the same time - after all - its easter - and i am not supposed to leave the house (apparently).

I prb wont do much on this for 24 hours (only as I need to solve my immediate issue) and will focus on my real job (which has nothing to do with programming) for a day or two ..   Will bake this for you properly no later than Monday night.


I didn't have my environment setup to run the full tests - this was originally intended to be a quick change :)   But all good I fixed that last night 

I will sort out the dispatch

Reply:
wow, @aidanmountford this is awesome! Thanks for helping out in the project and please let me know if there is something I can do to assist you.

I will let you know if I can think of anything else that could be added here.